src/components/accession2/view/Accession2View.tsx

```diff
@@ -45,6 +50,18 @@ const useStyles = makeStyles((theme: Theme) => ({
   fullSizeButton: {
     width: '100%',
   },
+  backIcon: {
+    fill: theme.palette.TwClrIcnBrand,
+    marginRight: theme.spacing(1),
+  },
+  back: {
```

Comment:
fontSize 20px? inventory page has that fontsize in the breadcrumb.
nit: rename `back` to something more descriptive? backToAccessions or accessionsBreadCrumb or anything else

Reply:
@charlie-tf will this be covered by the Typography visual updates?
If not, we should ask Ming/Chudo and agree on one font size for the bread-crumb.

Reply:
based on visual updates this will soon be 14px :)
if you want to keep it consistent until then - make it 20px and add a comment in the design for Ming (or change inventory to 16px).. up to you , if you prefer to stick to figma that's fine